lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -637,6 +637,11 @@ private int updateLastVersionsInData(String indexName, GUID parentGUID,
         if (indexName == null) {
```

Comment:
Also, indexObjects does a refresh as its last operation, which is now unnecessary due to the updateVersion refresh.

Reply:
Removed.